engine/engine.py

```diff
@@ -87,7 +87,27 @@ class NewEngineNodeTree(bpy.types.Operator):
     bl_label = "New Node Tree"
 
     def execute(self, context):
-        bpy.ops.node.new_node_tree(type="DreamTexturesNodeTree")
+        # TODO: Get the name of the default node tree from a config file
+        # TODO: type is deprecated https://docs.blender.org/api/current/bpy.types.NodeTree.html 
+        #       When testing the type of the resulting node tree using  bpy.data.node_groups['Dream Textures Node Tree'].type it is '' because of that
+        bpy.ops.node.new_node_tree(type="DreamTexturesNodeTree", name="Dream Textures Node Editor")
+        # Get the newly generated node tree
+        node_tree = bpy.data.node_groups[-1]
+        node_sd = node_tree.nodes.new(type="dream_textures.node_stable_diffusion")
```

Comment:
It might be worthwhile to pass `NodeStableDiffusion.bl_idname` as the type in case the ID changes in the future.